NWEEI/NWEEI/Views/About/History.cshtml

```diff
@@ -0,0 +1,77 @@
+﻿@{
+    ViewData["Title"] = "Mission & History";
+}
+<div class="container content-section">
+    <h3>Mission & History</h3>
+    <p>
+        The Northwest Water & Energy Education Institute (NWEEI) offers both standard and custom learning opportunities throughout the Northwest for energy and water industry practitioners – including an energy management certification program. NWEEI is located within the Science Department at Lane Community College.  Starting in 2018 all Energy Classes have been offered online and in Fall of 2019 the Water Conservation Program is also online.  Great opportunities for folks outside of the Lane Community College area.
+    </p>
+</div>
+
+<div class="dark-block">
+    <div class="container">
+        <div class="content-section">
```

Comment:
What would you think about adding a "History" heading here, where the dark block starts? The intro blurb above could be "Mission", since it's more NWEEI objective-based. I think it could add a nice flow from one section to the next.